src/main.ts

```diff
@@ -237,7 +238,14 @@ async function activateSdkExtension(eventStream: EventStream) {
         // Ensure SDK is installed.
         eventStream.post(new PackageInstallation(".NET 5.0 SDK"));
         const aquisitionResult = await vscode.commands.executeCommand<{ dotnetPath: string }>('dotnet-sdk.acquire', { version: '5.0' });
-        const result = aquisitionResult.dotnetPath?.length > 0
+
+        const success = aquisitionResult.dotnetPath?.length > 0;
+        if (success) {
+            process.env.PATH += path.delimiter + path.dirname(aquisitionResult.dotnetPath);
```

Comment:
I don't think this is going to help. You would need this change also in:
* The process that launches task.json tasks
* The debugger process

Reply:
It does work well enough for the processes launched by the C# extension, including OmniSharp, dotnet, and the debugger. It falls apart with the process we do not start such as the terminal and prelaunch tasks.

I am able to debug a pre-built .net core application from my first run of VS Code in the Sandbox.
![image](https://user-images.githubusercontent.com/611219/109568696-1f16b200-7a9c-11eb-87d5-fd8f399fec42.png)